### ISSUE TITLE
Add events to the inventory whenever a `Consumable` is consumed

### ISSUE THREAD

**Issue by Ishadijcks** [Ishadijcks/MooLite#27]
Add an event for food and drink separately. 

Make sure to add a notify option turned off when called from client info.

**Comment by calebanthony** [comment]
This should probably involve a refactor of updateCharacterDrink and updateCharacterFood in the Inventory class. Right now, it just takes the new object and overwrites the old one. Ideally, we're treating those methods similarly to updateCharacterItems where we're looping over each thing and dispatching events when the duration of the CharacterConsumable changes.

**Comment by Ishadijcks** [comment]
Indeed! You can take a look at `updateCharacterAbilities()` for an example. It also does some bookkeeping instead of overriding directly
